src/extern-sync/Tasks/SyncBoreholesTask.cs

```diff
@@ -77,10 +77,12 @@ protected override async Task RunTaskAsync(CancellationToken cancellationToken)
             publishedBorehole.Workgroup = null;
             publishedBorehole.WorkgroupId = targetWorkgroup.Id;
 
-            // Set publication status
-            publishedBorehole.SetBoreholePublicationStatusPublished();
-            publishedBorehole.Workflows.UpdateAttachedUser(targetDefaultUser);
-            publishedBorehole.Workflow?.ClearAssignedUser();
+            // Remove all previous workflow changes (= history).
+            publishedBorehole.Workflow.Changes.Clear();
```

Comment:
Q: Der Status 'publiziert' auf der Bohrung bleibt hier beim Sync erhalten oder?
Q: Gibt es in den Unterkategorien (Hydrogeology, Completion, etc) auch noch eine History?

Reply:
Q1: Stimmt das müsste vermutlich wieder auf DRAFT gesetzt werden, wenn die Daten von Prod Ext kommen. Oder wie seht ihr das @flenny @danjov  ?
Q2: Nein die gibts nur bei den Workflows.

Reply:
@MiraGeowerkstatt Das [bisherige Verhalten](https://github.com/swisstopo/swissgeol-boreholes-suite/blob/a75aafa89f074e97f87965d600288648a52c79eb/src/extern-sync/SyncContextExtensions.cs#L145-L161) war so, dass die Bohrungen als `Published` _gesynced_ wurden. Das würde ich beibehalten und den Status nicht auf DRAFT setzen.

Reply:
Ja der Status muss bleiben... wollte ich nur sicherstellen 😅